packages/app/src/vis-packs/core/complex/ComplexLineToolbar.tsx

```diff
@@ -53,26 +71,24 @@ function ComplexLineToolbar() {
 
       <Separator />
 
+      <Selector
+        value={visType}
+        onChange={(value: ComplexLineVisType) => setVisType(value)}
+        options={[ComplexVisType.Amplitude, ComplexVisType.Phase]}
+        optionComponent={({ option }) => (
+          // eslint-disable-next-line react/jsx-no-useless-fragment
+          <>{`${VIS_TYPE_SYMBOLS[option]} ${option}`}</>
+        )}
+      />
+
+      <Separator />
+
       <ToggleBtn
         label="Auto-scale"
         icon={MdDomain}
         value={autoScale}
         onToggle={toggleAutoScale}
-        disabled={isAutoScaleDisabled}
-      />
-
-      <ToggleBtn
-        label="Errors"
-        icon={(props) => (
-          <FiItalic
-            transform="skewX(20)"
-            style={{ marginLeft: '-0.25em', marginRight: '0.0625rem' }}
-            {...props}
-          />
-        )}
-        value={!areErrorsDisabled && showErrors}
-        onToggle={toggleErrors}
-        disabled={areErrorsDisabled}
```

Comment:
The NeXus complex line vis cannot show errors like the NeXus line vis can, so the button was always disabled.